### DISCUSSION TITLE
What is Zod's Website GitHub Repository Link

### DISCUSSION THREAD

**Discussion by babakfp** [colinhacks/zod#2945] [Q&A]
Hi
I want to do some fixes to https://zod.dev but I couldn't find the repo! Can someone send the link here?
Thanks.

**Comment by JacobWeisenburger** [comment]
I think it is here:
https://github.com/colinhacks/zod/blob/master/index.html

**Reply by babakfp** [reply]
That looks like an auto-generated file 🤔

**Reply by JacobWeisenburger** [reply]
interesting, then I'm not really sure
